cabal.project

```diff
@@ -0,0 +1,2389 @@
+packages:
+-- local packages
+    ./oscoin.cabal
+-- remote packages
```

Comment:
Does this clone the repos multiple times?

Reply:
you mean whether or not the use of multiple `subdir` would result in multiple clones? That didn't seem to be the case, but I wouldn't know, on top of my head (I was busy tried to patch up stuff with `hpack` that didn't pay attention to this detail!)

Reply:
Yeah it seems to not to. Bit of a nuisance that `subdir` is of type `Maybe FilePath` and not `[FilePath]`.